enterprise/server/remote_execution/commandutil/commandutil_test.go

```diff
@@ -284,21 +284,21 @@ func TestRun_EnableStats_ComplexProcessTree_RecordsStatsFromAllChildren(t *testi
 	}}
 
 	ctx := context.Background()
-	ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
+	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
 	defer cancel()
```

Comment:
tbh I'm not sure why I added this timeout, this LGTM but we can also just remove this and let the default test timeout do its thing.